ui/src/main/webapp/adminTools/webContextPolicyManager/styles.less

```diff
@@ -59,18 +59,14 @@
 
 .editPaneStyle {
   position: absolute;
-  top: 0px;
-  right: 0px;
-  bottom: 0px;
-  left: 0px;
-  padding: 10px;
+  right: 10px;
+  top: 10px;
   opacity: 0;
-  transition: opacity 0.5s ease;
+  transition: opacity .5s ease;
 }
 
-.editPaneStyle:hover {
+div:hover > .editPaneStyle {
```

Comment:
This causes the given style to be applied to the first child on hover if it has the property "editPaneStyle"